### ISSUE TITLE
Mac: Shortcut for Outline (⌃⌘O) doesn't work

### ISSUE THREAD

**Issue by gandalfsaxe** [typora/typora-issues#1424]
Does the shortcut for View->Outline (⌃⌘O) work for anyone else?

macOS 10.13.4 (17E199)
Typora 0.9.9.14

**Comment by AlexPasternak** [comment]
You're right, appears broken. It just creates new lines in a test document, even though clicking menu item for it correctly opens and closes the outline.
